transport/src/main/java/io/netty/channel/local/LocalChannel.java

```diff
@@ -24,6 +24,7 @@
 import io.netty.channel.ChannelPromise;
 import io.netty.channel.DefaultChannelConfig;
 import io.netty.channel.EventLoop;
+import io.netty.channel.PreferHeapByteBufAllocator;
```

Comment:
It doesn't look like it was "moved", but instead "copied" and there are 2 of them now. I'm wondering if we can get rid of the old one (`io.netty.channel.local.PreferHeapByteBufAllocator`) which no longer appears to be used.